erdgo/cmd/testTransaction/walletKey.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY for erd18yddfyzyskajjteyj90ueackpcsp9set2ma8vp54cj4zh54hjussukc6mu-----
```

Comment:
Perhaps use (copy) Alice's PEM instead (instead of having a new one) - optional?

https://github.com/ElrondNetwork/elrond-sdk/tree/master/erdpy/testnet/wallets/users

Reply:
Done